### PR TITLE
document how to provide and use mulitple VEE Ports

### PR DESCRIPTION
**Pull request by denisbedin**
https://docs.microej.com/en/feature-microej-sdk-6-multiple-vee-ports/

### CODE REVIEW

File: SDK6UserGuide/selectVeePort.rst [hunk 1]
```diff
@@ -7,16 +7,16 @@ Building or running an Application or a Test Suite with the SDK requires a VEE P
 
 There are 4 different ways to provide a VEE Port in the build file of the project:
 
-- Set the build property ``veePortFile`` in the ``microej`` configuration block to the path of a VEE Port file (``.zip``, ``.jpf``, or ``.vde``)::
+- Set the build property ``veePortFiles`` in the ``microej`` configuration block to the path of a VEE Port file (``.zip``, ``.jpf``, or ``.vde``)::
 
    microej {
-     veePortFile = "C:\\path\\to\\my\\veePort\\file.zip"
+     veePortFiles = listOf("C:\\path\\to\\my\\veePort\\file.zip")
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/selectVeePort.rst:13]: The section looks good, except that it is weird to use `listOf` here without explanation. Either we merge the sections (we directly explain that we can define one or more VEE Ports), or we add a sentence on the usage of `listOf` with a redirection to the next section. WDYT?

File: SDK6UserGuide/selectVeePort.rst [hunk 2]
```diff
@@ -50,23 +50,47 @@ There are 4 different ways to provide a VEE Port in the build file of the projec
...
+.. _sdk_6_select_multiple_vee_ports:
+
+Select multiple VEE Ports
+-------------------------
+
+It is possible to provide multiple VEE Ports to a project using one or more of 
+the :ref:`existing ways <sdk_6_select_veeport>` to provide a VEE Port. Each VEE Port will be used by the project.
+
+For example, you can set the ``veePortDirs`` to the path of a ``source`` folder of a VEE Port, 
+declare 2 dependencies with the ``microejVeePort`` conf and copy an archive file to the ``dropins`` folder.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/selectVeePort.rst:71]:
```suggestion
If multiple :ref:`VEE Ports are defined <sdk_6_select_veeport>` for a project, they are all used.
For example, you can set the ``veePortDirs`` to the path of a ``source`` folder of a VEE Port, 
declare 2 dependencies with the ``microejVeePort`` configuration and copy an archive file to the ``dropins`` folder.
```

File: SDK6UserGuide/selectVeePort.rst [hunk 3]
```diff
@@ -50,23 +50,47 @@ There are 4 different ways to provide a VEE Port in the build file of the projec
...
+
+To add several VEE Ports using the ``veePortFiles`` or the ``veePortDirs`` property, the paths must be separated by a comma. 
+For example ::
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/selectVeePort.rst:74]:
```suggestion
For example::
```

File: SDK6UserGuide/selectVeePort.rst [hunk 4]
```diff
@@ -50,23 +50,47 @@ There are 4 different ways to provide a VEE Port in the build file of the projec
...
+   microej {
+     veePortFiles = listOf("C:\\path\\to\\my\\veePort1\\file.zip","C:\\path\\to\\my\\veePort2\\file.zip")
+   } 
+
+.. _sdk_6_vee_port_unique_name:
+
+VEE Port unique name
+--------------------
+
+A VEE Port that has been provided has a unique name ``{name}-{toolchain}-{version}``,
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/selectVeePort.rst:85]:
```suggestion
VEE Ports are referenced by a unique name following the pattern ``{name}-{toolchain}-{version}``, 
```

File: SDK6UserGuide/selectVeePort.rst [hunk 5]
```diff
@@ -50,23 +50,47 @@ There are 4 different ways to provide a VEE Port in the build file of the projec
...
+.. _sdk_6_vee_port_unique_name:
+
+VEE Port unique name
+--------------------
+
+A VEE Port that has been provided has a unique name ``{name}-{toolchain}-{version}``, 
+where ``name``, ``toolchain`` and ``version`` values are the ones defined in the ``release.properties`` file of the VEE Port. 
+For example : `Platform-CM7hardfp_GCC48-0.1.0`
+
+If multiple VEE Ports have been provided, this name can be used to select the VEE Port on which
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/selectVeePort.rst:89]:
```suggestion
If multiple VEE Ports are defined, this name can be used to select the VEE Port on which 
```

File: SDK6UserGuide/runOnSimulator.rst [hunk 6]
```diff
@@ -99,8 +98,77 @@ If you want to connect the Eclipse debugger:
...
+     Eclipse runOnSimulator task Project Settings tab
+
+- Click on ``Run``
+  
+If the ``veePort`` property is not set, the task fails and an error message with the command line to use for each VEE Port is displayed::
+
+   > More than one VEE Ports have been defined. 
+     A VEE Port must be selected by defining the veePort property. Use one of the following commands to run the Application on the Simulator:
+  	    ./gradlew runOnSimulator -PveePort="veePort1"
+  	    ./gradlew runOnSimulator -PveePort="veePort2"
```
[the remark this answers is not in the document]
Comment (denisbedin) [reply SDK6UserGuide/runOnSimulator.rst]: Exact, done.

File: SDK6UserGuide/selectVeePort.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/selectVeePort.rst:13]: I added the sentence :
```
The ``veePortFiles`` property is defined as a list in order to provide multiple VEE Port files if it is needed.
See the :ref:`Select multiple VEE Ports <sdk_6_select_multiple_vee_ports>` section for more information.
```
Idem for the ``veePortDirs`` property. I think it is better to have a dedicated section for the multiple VEE Ports.

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**